v20.2/drop-type.md

```diff
@@ -0,0 +1,138 @@
+---
+title: DROP TYPE
+summary: The DROP TYPE statement drops an enumerated data type from the database.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> The `DROP TYPE` [statement](sql-statements.html) drops a specified [enumerated data type](enum.html) from the database.
+
+{{site.data.alerts.callout_info}}
+You cannot drop a type that is in use by a table.
+{{site.data.alerts.end}}
+
+## Required privileges
+
+The user must have the `DROP` [privilege](authorization.html#assign-privileges) on the parent database.
```

Comment:
cc @RichardJCai I'm not sure this is the case. Is it that the user must be the owner of the type?

Reply:
Yep, owner or member of the owner role.

Also considering adding DROP privilege on the type itself, not sure yet.